README.md

```diff
@@ -1 +1,60 @@
+
+The ultimate goal of this workflow is the optimisation of the proposal management of the Narayana project, in the effort of achieving a smoother transition of new RFEs from conceptualisation to realisation while fostering collaboration and knowledge sharing in our community.
+
+## 0. Secure the approval
+
+One of the fundamental improvements introduced by this workflow is the implementation of a process to secure the approval of the Narayana team (which consists of the [Contributors](https://www.narayana.io/community/index.html) in “Our team”, who are responsible for maintaining the Narayana code base). To progress a proposal, the whole Narayana team is required to approve it within a defined time window, which is set to 14 days[^1]. Optional members - specifically [Members](https://github.com/orgs/jbosstm/people?query=role%3Amember) and [Outside Contributors](https://github.com/orgs/jbosstm/outside-collaborators) - have the flexibility to participate at their discretion and their opinion is equal in importance to the [Narayana team](https://www.narayana.io/community/index.html). As a consequence, every contributor from the aforementioned groups (hereafter “reviewers”) can veto the approval of a proposal within the defined time window.
+
+When reviewers are analysing a proposal, there might be different outcome, which should be addressed in different ways:
+* All participating reviewers approve the proposal -> The proposal moves to the next stage
+* All participating reviewers reject the proposal -> The proposal is rejected, therefore it will be closed together with the related JIRA and pull request
+* The majority of participating reviewers approves the proposal -> The proposal needs to be reworked based on the feedback received. Participating reviewers need to provide a thorough feedback explaining why the proposal was rejected and how the author can improve the proposal to reach a satisfactory level to gain their approval
+* The majority of participating reviewers rejects the proposal -> The proposal is rejected. The participating reviewers and the author will consider whether any part part of the proposal can be improved and/or incorporated into a new proposal
+Community contributors are encouraged to present proposals and to share their opinions on any Request For Enhancement (RFE). However, any approval throughout the workflow must be granted by the Narayana team and its optional members.
+
+[^1]: The time window can be adjusted according to individual and team circumstances. However, the Narayana team is required to provide a different time window within 3 days from the date a new proposal is created.
```

Comment:
I think the first part of the footnote should remain but I think we should consider to remove `However, the Narayana team is required to provide a different time window within 3 days from the date a new proposal is created.` I think the 14 days should be the goal and then adjusting the time window as necessary would be complete enough for most cases.

Reply:
I'll remove the 3 days. Thanks Tom

Reply:
Thank you!